### PR TITLE
[Hotfix] News components medium screen layout improvement

### PR DESCRIPTION
**Pull request by kilemensi**
##### What does this PR do?

Improves **News** page components (`Monthly Stories` and `Video & Audio Stories`) on `md` screen sizes.

##### Description of Task to be completed?

 - [x] Separate `lg` from `md` styles, and
 - [x] Introduce `md` sizes using `%`

###### Medium screens

m

###### Extra large screens

x

### CODE REVIEW

File: src/components/Section.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    PropTypes.arrayOf(PropTypes.node),
+    PropTypes.node
+  ])
+};
+
+Section.defaultProps = {
+  children: null
+};
+
+export default withStyles(styles)(Section);
```
Comment (karimkawambwa) [inline src/components/Section.js:43]: We should think of ways to reuse this, the only problem the pages are really asymmetric, we would have to make ur own alignment decisions. 👍

File: src/components/MonthlyStories/StoryOfTheMonth.js [hunk 2]
```diff
@@ -32,12 +31,12 @@ const styles = theme => ({
...
     [theme.breakpoints.up('md')]: {
-      width: '39.375rem',
-      height: '30rem'
+      width: '50%',
+      height: '87.7%'
     }
   },
   story: {
-    width: '90%',
+    width: '88.9%',
```
Comment (karimkawambwa) [inline src/components/MonthlyStories/StoryOfTheMonth.js:39]: 😆 `0.1%` pixel perfection

File: src/components/MonthlyStories/Stories.js [hunk 3]
```diff
@@ -35,15 +44,17 @@ function Stories({ classes, stories }) {
...
           <NewsCard
             key={story.title}
             title={story.title}
             image={story.image}
             date={story.date}
             link={story.link}
             classes={{
-              root: classNames({ [classes.newsCardMargin]: index > 0 })
+              root: classNames(classes.newsCard, {
+                [classes.newsCardMargin]: index > 0
```
Comment (karimkawambwa) [inline src/components/MonthlyStories/Stories.js]: What happened to good ol `:first-child` 😢  ?

**Review by karimkawambwa — APPROVED** [review]
Beautiful 🚀 

----

You can make the `:first-child` change if you feel like it.

File: src/components/MonthlyStories/Stories.js (hunk 3 above)
Comment (kilemensi) [reply src/components/MonthlyStories/Stories.js]: When you write code in the flow _without_ taking a minute to think about it... This is what happens @karimkawambwa.

**Pull request merged**